### ISSUE TITLE
Provide ARIA functionality into Create a Dev sand box resource

### ISSUE THREAD

**Issue by xbabalov** [containers/podman-desktop#4361]
### Is your enhancement related to a problem? Please describe

Enhance ARIA accessibility of Developer Sandbox resources creation in Podman Desktop.

### Describe the solution you'd like

Provide ARIA accessibility to the Create a Developer Sandbox page

**Comment by deboer-tim** [comment]
This should be moved to redhat-developer/podman-desktop-sandbox-ext repo.

**Comment by dgolovin** [comment]
UI is generated by podman-desktop, extensions provides display name, ids (where appropriate) and etc, but don't contribute custom UI.

**Comment by deboer-tim** [comment]
Yeah, my bad... but that also means this is a generic request for all Kube providers (or none/something else entirely).
